frontend/packages/signless-transactions/src/hooks/use-create-session.ts

```diff
@@ -127,7 +131,9 @@ function useCreateSession(programId: HexString, metadata: ProgramMetadata | unde
     console.log('metadata.types?.others?.output');
     console.log(metadata.types?.others?.output);
 
-    const hexToSign = metadata.createType(metadata.types.others.output, payloadToSign).toHex();
+    const hexToSign = createSignatureType
```

Comment:
this is a solution for flexibility of approach. 
For example in `car races` it was necessary to have a type tuple for `metadata.types.others.output`, therefore we need to get only the first element of the array

Reply:
Good solution, let's leave a comment to describe the purpose of this prop.